code/modules/power/supermatter/supermatter_delamination.dm

```diff
@@ -193,61 +213,102 @@
 /datum/supermatter_delamination/proc/pick_rift_location()
 	var/turf/rift_location = get_turf(pick(GLOB.generic_event_spawns))
 	cascade_rift = new /obj/cascade_portal(rift_location)
+	message_admins("Exit rift created at [get_area_name(rift_location)]. [ADMIN_JMP(cascade_rift)]")
+	log_game("Bluespace Exit Rift was created at [get_area_name(rift_location)].")
+	cascade_rift.investigate_log("created at [AREACOORD(rift_location)].", INVESTIGATE_ENGINE)
```

Comment:
Investigate logs have area coordinates by default:
`14:08:21 [0x200a793] (169,154,2) || the supermatter crystal has been hit by the kiss blown by Alvin Allen fired by Projanitor/(Alvin Allen)`

Double coordinate logging:
`14:15:06 [0x200ad93] (175,150,2) || the emitter regained power and turned ON at Supermatter Engine Room (175,150,2)`

Reply:
oop